jiraworklog/src/Hoc/input.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Input } from 'semantic-ui-react-form-validator';
+
+const InputHoc = (props) => {
```

Comment:
what is this HOC? seems tightly coupled with Email field? Discuss with chetan quickly .. how we are doing it for NS